phys/module_surface_driver.F

```diff
@@ -4045,6 +4059,27 @@ SUBROUTINE surface_driver(                                         &
 
       ENDIF
 
+      DO j=j_start(ij),j_end(ij)                                               
+         DO i=i_start(ij),i_end(ij)
+            CHKLOWQ(I,J)= 1.0
+            SFCEVP(I,J)= SFCEVP(I,J) + QFX(I,J)*DTBL
+            !SFCEXC(I,J)= CHS(I,J)
+            IF(PRESENT(ACHFX))ACHFX(I,J)=ACHFX(I,J) + HFX(I,J)*DT
+            IF(PRESENT(ACLHF))ACLHF(I,J)=ACLHF(I,J) + LH(I,J)*DT
+            IF(PRESENT(ACGRDFLX))ACGRDFLX(I,J)=ACGRDFLX(I,J) + GRDFLX(I,J)*DT
+         ENDDO
+      ENDDO
+
+        !CALL SFCDIAGS(HFX,QFX,TSK,QSFC,CHS2,CQS2,T2,TH2,Q2,      &
+        !           PSFC,CP,R_d,RCP,CHS,t_phy,qv_curr,ua_phys,    &
+        !           ids,ide, jds,jde, kds,kde,                    &
+        !           ims,ime, jms,jme, kms,kme,                    &
+        !            i_start(ij),i_end(ij), j_start(ij),j_end(ij), kts,kte    )
```

Comment:
@negin513 
If these are commented out, let's just remove them.

Reply:
@davegill thanks for your comment. I just removed these commented out lines in the recent commits. 
The reason I kept it there in the first place was to keep it consistent with other LSMs. For example, for NOAH-MP these lines are just commented out and not removed: 
https://github.com/wrf-model/WRF/blob/ca7092a948d327104eb0ae3b0224a70f3b21c2bc/phys/module_surface_driver.F#L3306-L3312
But, I agree with you that removing them makes more sense.

Reply:
I am going ahead to resolve this.